connection.go

```diff
@@ -230,6 +230,19 @@ type connShard struct {
 	enc             *msgpack.Encoder
 }
 
+// RLimitActions is an enumeration type for an action to do when a rate limit
+// is riched.
+type RLimitAction int
+
+const (
+	// RLimitDrop immediately aborts request,
```

Comment:
````suggestion
        // RLimitDrop immediately aborts the request.
```
(I don't suggest any indentation changes, GitHub just doesn't support tabs in code blocks)

Reply:
Updated.